client/src/components/generic/DebounceTextField.js

```diff
@@ -17,7 +17,7 @@ export const DebounceTextField = ({ time, onDebounce, onChange, ...props }) => {
 
   const handleOnChange = (element) => {
     setText(element.target.value);
-    onChange(element);
+    onChange && onChange(element);
```

Comment:
Are there issues with onChange not being defined?

Reply:
Yes, I've same question

Reply:
This component handle it's own textfield state so the onchange/value props aren't really needed. I removed those values from the implementations of this component and only use `defaultValue`/`onDebounce` props. Removing `onChange` cause an error here